dbms/src/Storages/DeltaMerge/DeltaMergeStore.h

```diff
@@ -353,7 +353,10 @@ class DeltaMergeStore : private boost::noncopyable
                               const SegmentIdSet & read_segments = {},
                               size_t extra_table_id_index = InvalidColumnID);
 
-    /// Read rows with MVCC filtering
+
+    /// Read rows in two modes:
+    ///     when is_raw_read == true, read rows witch MVCC filtering
+    ///     when is_raw_read == false, read rows without MVCC and sorted merge
```

Comment:
We need more explanation to clarify the difference between `readRaw` and `read` with `is_raw_read==true`
They are confusing now...